timetracker/timetracker/settings.py

```diff
@@ -20,12 +20,12 @@
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'fex(9l-^pfdvrzbd&&5!(f3cges$o#5o^sv=e#bbof*es$$*bl'
+SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', 'ThIs-K3Y-iSnT-a-5eCrEt')
```

Comment:
I'd recommend to go with `None` as a default to make sure you have one set in production.